internal/elasticsearch/logstash/pipeline.go

```diff
@@ -134,15 +133,10 @@ func ResourceLogstashPipeline() *schema.Resource {
 			Type:        schema.TypeBool,
 			Optional:    true,
 		},
-		"queue_max_bytes_number": {
-			Description: "The total capacity of the queue when persistent queues are enabled.",
-			Type:        schema.TypeInt,
-			Optional:    true,
-		},
-		"queue_max_bytes_units": {
+		"queue_max_bytes": {
 			Description:  "Units for the total capacity of the queue when persistent queues are enabled.",
 			Type:         schema.TypeString,
-			ValidateFunc: validation.StringInSlice([]string{"b", "kb", "mb", "gb", "tb", "pb"}, false),
+			ValidateFunc: validation.StringInSlice([]string{"2b", "2kb", "2mb", "2gb", "2tb", "2pb"}, false),
```

Comment:
@yaseen-nbs - Forgive me if I'm wrong but will this not just limit potential values to having a value of 2? e.g. will this not error if you set the test to 5gb instead or another value other than 2

Reply:
that's how it is defined the parameters for string option? I can change it back if that confusing. thanks.

Reply:
So previously the number and units were split, so it's easier to just validate the number is an int and that the units match one of the strings defined in the validation slice (b, kb etc.). As the number field and unit values are now both being combined into a string, you'd need to find a way to validate that the string matches some pattern ([0-9]b, [0-9]kb or something similar), but I don't think validation.StringInSlice will allow for you to regex match against the string and literally matches so you may need to find a different method / approach for applying this input validation 👍